tests/test_feedexport.py

```diff
@@ -443,7 +443,160 @@ def test_export_encoding(self):
             'csv': u'foo\r\nTest\xd6\r\n'.encode('latin-1'),
         }
 
-        for format in formats:
-            settings = {'FEED_FORMAT': format, 'FEED_EXPORT_ENCODING': 'latin-1'}
+        settings = {'FEED_EXPORT_INDENT': None, 'FEED_EXPORT_ENCODING': 'latin-1'}
+        for format, expected in formats.items():
+            settings['FEED_FORMAT'] = format
             data = yield self.exported_data(items, settings)
-            self.assertEqual(formats[format], data)
+            self.assertEqual(expected, data)
+
+    @defer.inlineCallbacks
+    def test_export_indentation(self):
+        items = [dict({'foo': ['bar']})]
+
+        output = [
+            # JSON
+            {
+                'format': 'json',
+                'indent': None,
+                'expected': b'[\n{"foo": ["bar"]}\n]',
+            },
+            {
+                'format': 'json',
+                'indent': -1,
+                'expected': b"""
+[
+{
+"foo": [
+"bar"
+]
+}
+]
+""",
+            },
+            {
+                'format': 'json',
+                'indent': 0,
+                'expected': b"""
+[
+{
+"foo": [
+"bar"
+]
+}
+]
+""",
+            },
+            {
+                'format': 'json',
+                'indent': 2,
+                'expected': b"""
+[
+{
+  "foo": [
+    "bar"
+  ]
+}
+]""",
```

Comment:
hm, is this correct that here there is a newline in the beginning on the file instead of a newline at the end, and with indent=0 there are new lines both in the beginning and in the end? Wouldn't it be better not to have a new line in the beginning and always have a new line in the end (unless indent is None)?